hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/validators/block/TransactionRecordParityValidator.java

```diff
@@ -85,7 +85,7 @@ public void validateBlockVsRecords(@NonNull final List<Block> blocks, @NonNull f
 
         final var actual = translateAll(inputs);
 
-        final var maxDiffs = 10;
+        final var maxDiffs = 1000;
```

Comment:
I increased the number because I am looking at the diffs from the whole `SubmitMessageSuite` and there were a lot of CryptoCreate ones which are not the focus of the task which is to find and fix the `ConsensusSubmitMessage` ones. 

I can revert it to back to 10 but if any reviewer wants to verify if there are any `ConsensusSubmitMessage` diffs left to be fixed they will need to remember to increase this value otherwise it can be misleading and will look like there are no `ConsensusSubmitMessage` diffs to be fixed in the `SubmitMessageSuite` block stream.

Reply:
Reverted back to 10.

Reply:
Ahh I understand, didn't think about that. We should come up with a solution for this 🤔